pkg/operator/types.go

```diff
@@ -53,3 +53,31 @@ func MakeHostAliases(input []monitoringv1.HostAlias) []v1.HostAlias {
 
 	return output
 }
+
+const ShardLabel = "operator.prometheus.io/shard"
+const ResourceNameLabel = "operator.prometheus.io/name"
```

Comment:
Could we import the constants from the prometheus package instead? 

https://github.com/prometheus-operator/prometheus-operator/blob/79502954f995929c8b3511d533a2e34bb3ee40ca/pkg/prometheus/statefulset.go#L60-L61

Reply:
I tried but we got a cyclical dependency 😢